packages/ui/components/Home/Logos.vue

```
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+  const logos = [
+    'simple-icons:adobe',
+    'simple-icons:amazon',
+    'simple-icons:apple',
+    'simple-icons:google',
+    'simple-icons:meta',
+    'simple-icons:microsoft',
+    'simple-icons:shopify',
+    'simple-icons:slack'
+  ];
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)